src/messages.js

```diff
@@ -62,6 +62,20 @@ messages[ operations.MINUS ] = [
   }
 ];
 
+messages[ operations.EQUAL ] = [
+  {
+    probability: 100,
+    set: [
+      'How is',
+      'Why is'
+    ]
+  },
+  {
+    probability: 1,
+    set: [ ':shifty:' ]
```

Comment:
I see you've also joined the cult of the shifty. One of us! One of us! One of us!